### PR TITLE
Add support for company selection wrap around

### LINKED ISSUE #100: Setting "company-selection-wrap-around" with company-box-mode enabled has no effect.
I figured that the "company-selection-wrap-around" functionality from company-mode does no longer work when company-box-mode is enabled. In solely company-mode it works fine. Tested with Emacs 27 with native json. I can't seem to find any reference to this, whether its an issue or unsupported feature. Although it not a major problem, sometimes it gets annoying to scroll all the way up! :D

### PR DESCRIPTION
**Pull request by larebsyed**
**Before Fix** 
The selection doesn't wrap around when `company-selection-wrap-around` is set true
**After Fix**
It should wrap around as expected
Fixes #100

### CODE REVIEW

File: company-box.el [hunk 1]
```diff
@@ -682,8 +691,9 @@ COMMAND: See `company-frontends'."
   (setq company-box--prefix-pos nil
         company-box--start nil
         company-box--edges nil)
-  (company-box--set-frame-position (company-box--get-frame))
-  (company-box--update-scrollbar (company-box--get-frame) t))
+  (when (company-box--get-frame)
```
Comment (larebsyed) [inline company-box.el:694]:
```suggestion

```

File: company-box.el [hunk 2]
```diff
@@ -682,8 +691,9 @@ COMMAND: See `company-frontends'."
...
+    (company-box--set-frame-position (company-box--get-frame))
+    (company-box--update-scrollbar (company-box--get-frame) t)))
```
Comment (larebsyed) [inline company-box.el:696]:
```suggestion
    (company-box--update-scrollbar (company-box--get-frame) t))
```

**Comment by larebsyed** [comment]
@sebastiencs Please take a look

**Comment by sebastiencs** [comment]
Thanks @larebsyed, can you please resolve the conflict

**Comment by larebsyed** [comment]
@sebastiencs Done, Thank you for taking a look

**Pull request merged**